vspace/src/main/webapp/WEB-INF/views/exhibition/space.html

```diff
@@ -73,6 +102,29 @@
     <div layout:fragment="content">
         <script th:inline="javascript">
 		//# sourceURL=click.js
+		
+            let isResizeImgBtnClicked = 1;
+            function toggleImgSize(){
+                let maxImgWidth = '1200px';
```

Comment:
it should take the maximum possible width given window size (so not bigger than window size and as big as possible as long as it's not artificially blowing up the image).